ros2component/package.xml

```diff
@@ -9,16 +9,22 @@
   <maintainer email="michel@ekumenlabs.com">Michel Hidalgo</maintainer>
   <license>Apache License 2.0</license>
 
+  <depend>ament_index_python</depend>
```

Comment:
This shouldn't be needed a build time?

Same below - also for various dependencies not touched in this PR.

Reply:
Correct, I replicated below's bad practices. Fixed